src/PerformanceTable.h

```diff
@@ -0,0 +1,60 @@
+#ifndef PERFORMANCETABLE_H
+#define PERFORMANCETABLE_H
+
+#include "Criteria.h"
+#include "Performance.h"
+#include <iostream>
+#include <vector>
+
+class PerformanceTable {
+public:
+  /**
+   * Performances constructor with defined vector of performance
+   *
+   * @param perf_vect Vector of performance
+   */
+  PerformanceTable(std::vector<Performance> &perf_vect);
+
```

Comment:
Will we get a PT that will have a sort of matrix evaluation ? 
ie could we access pt[i][j] which is the performance of the j_th criteria of the i_th alternative ?
I don't if we can do that easily with this structure or we might need to create an [][] operator overload ?

Reply:
Yes, I'll add a [] operator to the PT to get access directly to a Performance vector, thus the [i][j] operator will access a performance of the j_th criteria of the i_th alternative, at a cost of O(n^2) given the implementation. However this huge access cost is not likely to be an issue in our case as it will not be a common case to access directly a Perf object from the PT.

Reply:
To implement the get_best, get_worst, get_middle, etc ... we will need to get the each column of the PT so we will need to access those elements so we will have a cost of O(n^2) ? I that's the case ( I think so), might need to think about other data types ?

Reply:
The datastructure was designed to be optimized to be itterated over all its content (for sorting algorithms for example), as it will not be a common case to access a single Perf directly. get_above, get_best and get_middle are particular functions that will be implemented in O(1) as we already know where the elements will be (first position, last position, middle position). If at some point accessing the value of a specific criteria of a specific alternative (pt[i][j]) become a common operation we will need to modify the datastructure to use unordered hashmap to get the access time in O(1) in the general case.